MediaPlayer/MediaPlayer.module.less

```diff
@@ -1,25 +1,3 @@
-@import "../styles/skin.less";
-
 .mediaPlayer {
-	.times {
-		display: flex;
-		justify-content: space-between;
-		position: relative;
-		bottom: 15px;
-	}
-
-	.applySkins({
-		height: 132px;
```

Comment:
Where is this go?

Reply:
Oupss, deleted by mistake. I added it back.